source/client/client_worker_impl.cc

```diff
@@ -69,7 +95,8 @@ void ClientWorkerImpl::shutdownThread() { benchmark_client_->terminate(); }
 StatisticPtrMap ClientWorkerImpl::statistics() const {
   StatisticPtrMap statistics;
   StatisticPtrMap s1 = benchmark_client_->statistics();
-  StatisticPtrMap s2 = sequencer_->statistics();
+  auto& sequencer = phases_.back()->sequencer();
```

Comment:
Can we prefer to mention the Sequencer type explicitly (instead of auto) unless it is too unwieldy?

Reply:
Done